src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php

```diff
@@ -41,69 +39,62 @@ final class DashboardController
     /** @var SalesDataProviderInterface|null */
     private $salesDataProvider;
 
+    /** @var StatisticsDataProviderInterface */
+    private $statisticsDataProvider;
+
     public function __construct(
-        DashboardStatisticsProviderInterface $statisticsProvider,
         ChannelRepositoryInterface $channelRepository,
         EngineInterface $templatingEngine,
         RouterInterface $router,
-        ?SalesDataProviderInterface $salesDataProvider = null
+        ?SalesDataProviderInterface $salesDataProvider = null,
+        ?StatisticsDataProviderInterface $statisticsDataProvider = null
     ) {
-        $this->statisticsProvider = $statisticsProvider;
         $this->channelRepository = $channelRepository;
         $this->templatingEngine = $templatingEngine;
         $this->router = $router;
         $this->salesDataProvider = $salesDataProvider;
-
-        if ($this->salesDataProvider === null) {
-            @trigger_error(
-                sprintf('Not passing a $salesDataProvider to %s constructor is deprecated since Sylius 1.7 and will be removed in Sylius 2.0.', self::class),
-                \E_USER_DEPRECATED
-            );
-        }
+        $this->statisticsDataProvider = $statisticsDataProvider;
     }
 
     public function indexAction(Request $request): Response
     {
-        $channelCode = $request->query->get('channel');
-
         /** @var ChannelInterface|null $channel */
-        $channel = $this->findChannelByCodeOrFindFirst($channelCode);
+        $channel = $this->findChannelByCodeOrFindFirst($request->query->get('channel'));
 
         if (null === $channel) {
             return new RedirectResponse($this->router->generate('sylius_admin_channel_create'));
         }
-        // this data will be getting from UI after improve graph on dashboard
-        $startDate = (new \DateTime('first day of next month last year'));
-        $endDate = (new \DateTime('last day of this month'));
-
-        $statistics = $this->statisticsProvider->getStatisticsForChannelInPeriod($channel, $startDate, $endDate);
-        $data = ['statistics' => $statistics, 'channel' => $channel];
-
-        if ($this->salesDataProvider !== null) {
-            // this data will be getting from UI after improve graph on dashboard
-            $interval = Interval::month();
-
-            $data['sales_summary'] = $this
-                ->salesDataProvider
-                ->getSalesSummary($channel, $startDate, $endDate, $interval)
-            ;
-            $data['currency'] = $channel->getBaseCurrency()->getCode();
+
+        return $this->templatingEngine->renderResponse('@SyliusAdmin/Dashboard/index.html.twig', [
+            'channel' => $channel,
+        ]);
+    }
+
+    public function getRawData(Request $request): Response
```

Comment:
```suggestion
    public function getRawDataAction(Request $request): Response
```

We tend to add `action` at the end of functions in controller. But this is a minor thing.